cinema/views.py

```diff
@@ -101,8 +104,26 @@ def get_serializer_class(self):
         if self.action == "retrieve":
             return MovieDetailSerializer
 
+        if self.action == "upload_image":
+            return MovieImageSerializer
+
         return MovieSerializer
 
+    @action(
+        methods=["POST"],
+        detail=True, url_path="upload-image",
+        permission_classes=[IsAdminUser]
```

Comment:
I will not reject it because of this, but if you write variables from a new line, then write all variables from a new line:
```python
methods=["POST"],
detail=True, 
url_path="upload-image",
permission_classes=[IsAdminUser]
```